lib/mail_room/mailbox_watcher.rb

```diff
@@ -114,6 +114,8 @@ def run
       @running = true
 
       self.idling_thread = Thread.start do
+        process_mailbox
```

Comment:
I felt like there was a reason I did it this way, but I don't remember now. Still, not sure how I feel about it, but in light of everything else, it's the least of my worries.